### DISCUSSION TITLE
xpack.security.session.concurrentSessions

### DISCUSSION THREAD

**Discussion by alveare1** [Security-Onion-Solutions/securityonion#12935] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 to 16 depending on node

### RAM

16 or 32gb

### Storage for /

512

### Storage for /nsm

512

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am running a distributed cluster and want to limit my users on sessions. I see this can be set on elastic and want to make sure I am not going to cause issues setting this in the kibana yml. I didn't see this option in the SOC configuration.

xpack.security.session.concurrentSessions:
    maxSessions:2

https://www.elastic.co/guide/en/kibana/current/xpack-security-session-management.html

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
I don't believe this will cause an issue, but bear in mind that this will only affect concurrent sessions in Kibana, not in SOC.
